recipe-server/normandy/base/genome.py

```diff
@@ -0,0 +1,136 @@
+import math
+import random
+import string
+
+from hashlib import sha256
+
+
+class Genome:
```

Comment:
I think this is a question for @mythmon.

Reply:
As long as the same guarantees are provided, I'm ok with this. Since we can seed Python's Random class, instead of JS's single global object, we can exert more control over than we could in JS. As long as the same calls in the same order always produce the same results (and we have tests for this), I'm ok with subclassing `Random`.

Reply:
Oh, and to clarify, the example above isn't how I think we should end up using the Random class (since it's not subclassing it like I think we should), just on how the objects created by the class work.